packages/terra-application-layout/docs/utils.md

```diff
@@ -0,0 +1,73 @@
+# Terra Application Layout
+
+In addition to the ApplicationLayout component, the package exports an `Utils` object that contains a variety of helper functions and prop type declarations. Included are:
+
+- `propTypes` - An object containing prop type definitions for the configuration objects used by the ApplicationLayout.
+- `helpers` - An object containing helper functions to aid in the creation of ApplicationLayout content.
+- `utilityHelpers` - An object containing helper functions specifically for the creation of Utility menu configuration.
+
+## `propTypes`
+
+- `layoutConifgPropType` - Shape for the `layoutConfig` prop provided to components within the ApplicationLayout.
+- `nameConfigPropType` - Shape for ApplicationLayout's `nameConfig` prop.
+- `navigationItemsPropType` - Shape for ApplicationLayout's `navigationItems` prop.
+- `utilityConfigPropType` - Shape for ApplicationLayout's `utilityConfig` prop.
+
+## `helpers`
+
+- `isSizeCompact` - A function that determines whether or not the given size is considered `compact` or not. This can be used to synchronize component rendering with responsive changes to the ApplicationLayout.
+
+### `utilityHelpers`
+
+- `getDefaultUtilityConfig`
+
+A function that generates an object conforming to the `utilityConfig` shape and containing the Terra-standard set of utility menu items.
+
+An `intl` parameter is required, as the translated titles for the default configuration must be retrieved from it. Also required is an object containing user information, as the default menu configuration is user-centric. Finally, additional menu configuration items can be provided to be merged with the default items
```

Comment:
is ~an~ **the `userData`** object

Reply:
> default items

1. What are the default items? Add a link maybe?
2. Missing period.

Reply:
I will list out the default items.